charts/k8ssandra-operator/crds/k8ssandra-operator-crds.yaml

```diff
@@ -10258,7 +10258,7 @@ spec:
                             ServerVersion is the Cassandra or DSE version. The following versions are supported:
                             - Cassandra: 3.11.X, 4.X.X and 5.X.X
                             - DSE: 6.8.X, 7.x.x
-                          pattern: (6\.8\.\d+)|(3\.11\.\d+)|(4\.\d+\.\d+)|(5\.\d+\.\d+)|(7\.\d+\.\d+)
+                            - HCD: 1.0.x
```

Comment:
suggestion: We shouldn't remove pattern entirely, it should be at least something like ``\d+\.\d+\.\d+``. We're not supporting versions like ``6.9`` that would auto-update to newest point-release for example.

Reply:
fair enough 👍